setup.py

```diff
@@ -1,34 +1,70 @@
 # Copyright 2017-2020 Lawrence Livermore National Security, LLC and other
-# Hatchet Project Developers. See the top-level LICENSE file for details.
+# CallFlow Project Developers. See the top-level LICENSE file for details.
 #
 # SPDX-License-Identifier: MIT
 
+import os
 from setuptools import setup, find_packages
-from callflow import __version__
 
+# ------------------------------------------------------------------------------
+# get the version safely!
+from codecs import open
+
+version = {}
+vfile = os.path.join(
+    os.path.dirname(os.path.abspath(__file__)), "callflow", "version.py"
+)
+with open(vfile) as fp:
+    exec(fp.read(), version)
+version = version["__version__"]
+
+# ------------------------------------------------------------------------------
+# gather the data to be copied
+def list_files(directory):
+    paths = []
+    for (path, directories, filenames) in os.walk(directory):
+        files = [os.path.join(path, f) for f in filenames]
+        paths.append((path, [os.path.join(path, f) for f in filenames]))
+    return paths
+
+
+data_files = list_files("data")
+example_files = list_files("examples")
+
+
+deps = [
+    "flask_socketio",
+    "ipython",
+    "colorlog",
+    "jsonschema",
+    "numpy",
+    "scipy",
+    "pandas",
+    "scikit_learn",
+    "statsmodels",
+    "hatchet",
+    "networkx",
+    "matplotlib",
+]
+# ------------------------------------------------------------------------------
+# now set up
 setup(
     name="CallFlow",
-    version=__version__,
+    version=version,
     license="MIT",
     description="",
     url="https://github.com/LLNL/CallFlow",
-    author="Suraj Kesavan, Huu Tan Nguyen",
+    author="Suraj Kesavan",
     author_email="spkesavan@ucdavis.edu",
     classifiers=[
         "Development Status :: 3 - Alpha",
         "License :: OSI Approved :: MIT License",
     ],
     keywords="",
     packages=find_packages(),
-    install_requires=[
-        "numpy",
-        "pandas",
-        "tables",
-        "flask",
-        "flask_socketio",
-        "sklearn",
-        "statsmodels",
-        "networkx == 2.2",
-        "hatchet",
-    ],
+    data_files=data_files + example_files,
```

Comment:
Can we call the function directly here to save up some variable memory?

Reply:
surre. but it will very likely not save you anything.  since the code still needs to call 2 functions, collect their results and then add into a third. doesnt matter if you name the variable or not. python does not do return value optimiizations like cpp.

Reply:
Yes, I just read about this. I wasn't aware of python only creating tags and not slots in memory. I like this, I will keep it this way.